src/platform/plugins/shared/data_view_editor/public/components/preview_panel/status_message/status_message.test.tsx

```diff
@@ -77,51 +95,57 @@ describe('StatusMessage', () => {
       partialMatchedIndices: [],
     };
 
-    const component = shallow(
+    renderWithI18n(
       <StatusMessage
+        isIncludingSystemIndices={false}
         matchedIndices={localMatchedIndices}
         query={'k'}
-        isIncludingSystemIndices={false}
         showSystemIndices={false}
       />
     );
 
-    expect(component).toMatchSnapshot();
+    expect(screen.getByTestId('createIndexPatternStatusMessage')).toHaveTextContent(
+      /The index pattern you entered doesn't match any data streams, indices, or index aliases. You can match 2 sources./
+    );
   });
 
   it('should show that system indices exist', () => {
-    const component = shallow(
+    renderWithI18n(
       <StatusMessage
+        isIncludingSystemIndices={false}
         matchedIndices={{
           allIndices: [],
           exactMatchedIndices: [],
           partialMatchedIndices: [],
           visibleIndices: [],
         }}
-        isIncludingSystemIndices={false}
         query={''}
         showSystemIndices={false}
       />
     );
 
-    expect(component).toMatchSnapshot();
+    expect(
+      screen.getByText('No data streams, indices, or index aliases match your index pattern.')
+    ).toBeVisible();
```

Comment:
Maybe at some point we actually did "show that system indices exist", but from the snapshots it looks like we no longer do and this test is outdated. I think we could probably remove it, but also fine to leave since it was already this way.